source/command-monitoring/command-monitoring.rst

```diff
@@ -250,7 +250,8 @@ value MUST be replaced with an empty BSON document. The list is as follows:
    * - ``copydbgetnonce``
    * - ``copydbsaslstart``
    * - ``copydb``
-   * - ``isMaster`` or ``ismaster`` when ``speculativeAuthenticate`` is present
+   * - ``hello`` or ``hello`` when ``speculativeAuthenticate`` is present
```

Comment:
This and the line after should just be combined. "hello or legacy hello...". You might have to deal with the camel case vs. lower case issue in your "legacy hello" definition elsewhere.

Reply:
I was going to suggest switching to a case-insensitive comparison here to cover this, then realised that the server only accepts `isMaster` and `ismaster`, but rejects `isMAster`. Are there any other cases where we might have to account for different capitalisation of the legacy hello command?

Reply:
Combined and re-worded those lines as well as adding a link to the handshake spec.